src/features/quest/ui/styles.ts

```diff
@@ -160,6 +160,26 @@ export const QuestsTitle = styled.h2<QuestUrlProps>`
     `}
 `;
 
+export const LoginRequiredMessage = styled.p<QuestUrlProps>`
+  ${({ $isLearn }) =>
+    $isLearn &&
+    css`
+      color: #000000;
+      font-size: 13px;
+      font-weight: 700;
+      text-align: center;
+      margin-top: 25px;
+    `}
+  ${({ $isQuest = true }) =>
+    $isQuest &&
+    css`
+      color: #000000;
+      font-size: 15px;
+      font-weight: 700;
+      text-align: center;
+    `}
+`;
```

Comment:
```suggestion
export const LoginRequiredMessage = styled.p<QuestUrlProps>`
  color: #000000;
  text-align: center;
  font-weight: 700;
  
  ${({ $isLearn }) =>
    $isLearn &&
    css`
      font-size: 13px;
      margin-top: 25px;
    `}
  ${({ $isQuest = true }) =>
    $isQuest &&
    css`
      font-size: 15px;
    `}
`;
```
text-align, color, font-weight 공통 스타일을 그냥 위에도 작성해도 좋을 것 같습니다 ㅎㅎ

Reply:
그렇네요! 감사합니다~~